cocotb/share/include/gpi_logging.h

```diff
@@ -113,7 +113,7 @@ typedef void (gpi_log_handler_type)(
     @param funcname  Name of the function where the call site is located
     @param lineno    Line number of the call site
     @param msg       The message to log, uses C-sprintf-style format specifier
-    @param ...       Additional arguments; formatted at inserted in message according to format specifier in msg argument
+    @param args      Additional arguments; formatted and inserted in message according to format specifier in msg argument
```

Comment:
Should the non-"vlog" functions keep `...` instead of `args`?

Reply:
Yep.